c/.gitignore

```diff
@@ -0,0 +1,15 @@
+# binaries
+hello
+
+# build
+*.[aod]
```

Comment:
```suggestion
```
when using cmake or meson with github's template, these are not necessary as we have a standard build directory and file extensions are already matched